.sync/Files.yml

```diff
@@ -436,6 +446,7 @@ group:
       microsoft/mu_feature_config
       microsoft/mu_feature_debugger
       microsoft/mu_feature_dfci
+      microsoft/mu_feature_ffa

```

Comment:
Any issues with the Visual Studio build in that repo?

Reply:
good catch. let's not add it for now.

Reply:
Removed.